api/internal/loctarget/locloader.go

```diff
@@ -21,15 +21,16 @@ type LocArgs struct {
 	// target; local copy if remote
 	Target filesys.ConfirmedDir
 
-	// directory that bounds target's local references, empty string if target is remote
+	// directory that bounds target's local references
+	// repo directory of local copy if target is remote
 	Scope filesys.ConfirmedDir
 
 	// localize destination
 	NewDir filesys.ConfirmedDir
 }
 
-// locLoader is the Loader for kustomize localize. It is an ifc.Loader that enforces localize constraints.
-type locLoader struct {
+// LocLoader is the Loader for kustomize localize. It is an ifc.Loader that enforces localize constraints.
+type LocLoader struct {
```

Comment:
Done. I initially named it `LocLoader` because I didn't want readers to construe it with the `ifc` Loader interface definition and the plugins Loader, but I see your point!